src/main/java/com/questhelper/panel/QuestHelperPanel.java

```diff
@@ -297,11 +301,16 @@ public void changedUpdate(DocumentEvent e)
 		JPanel orderPanel = makeDropdownPanel(orderDropdown, "Ordering");
 		orderPanel.setPreferredSize(new Dimension(PANEL_WIDTH, DROPDOWN_HEIGHT));
 
+		skillSelectionPanel = makeNewDropdown(QuestHelperConfig.SkillFilter.values(), "skillReward");
+		JPanel skillSortPanel = makeDropdownPanel(skillSelectionPanel, "Skill Filtering");
```

Comment:
I'd personally prefer if this was labeled `Skill Filters` (consistent with main Filters), or `Skill Filter`